### DISCUSSION TITLE
Is there an Enterprise level Dashboard for Dependabot, Code Scanning, and Secret Scanning?

### DISCUSSION THREAD

**Discussion by NadiaThePuggle** [community/community#59676] [Enterprise]
### Select Topic Area

Question

### Body

We recently started using GitHub Advanced Security Insights mostly for Dependabot and Secret Scanning across all our organizations and repos.  I can view dashboards that aggregate the totals and the details for each Organization as well as at the Repo level for each Organization, but I don't see a way to view the results aggregated at the Enterprise level comprising all of our orgs and repos.  Is this possible?

**Comment by aashah** [comment]
Does https://docs.github.com/en/enterprise-cloud@latest/code-security/security-overview/about-security-overview look like what you were looking for?

**Comment by blaze0089** [comment]
Yes this actually exists, it's called Security Overview and it's exactly what you're looking for.

It lets you see Dependabot, Code Scanning, and Secret Scanning results aggregated across all your orgs and repos at the Enterprise level, so you don't have to jump between each org manually.

You can check the full docs here:
https://docs.github.com/en/enterprise-cloud@latest/code-security/security-overview/about-security-overview

That should cover everything you need to get it set up and start viewing the enterprise-wide view.
